Dockerfile

```diff
@@ -24,7 +24,7 @@
 #     docker run -it --rm libmpc bash
 # ------------------------------------------------------------------------------
 
-FROM ubuntu:bionic
+FROM ubuntu:latest
```

Comment:
hi Kealan, we could indeed name a version, but using the tag `latest` has the advantage that it will always point to the latest LTS (and not the literal latest), and thus avoid accidentally ending up relying on unmaintained/insecure dependencies, while still enjoying the solidity of the LTS quality assurances. I would thus suggest to keep `latest`.

Reply:
OK That is fine by me